versionedRouter.js

```diff
@@ -83,6 +83,7 @@ async function handleDest(ctx, destHandler) {
   );
   logger.debug(`[DT] Output events: ${JSON.stringify(respList)}`);
   ctx.body = respList;
+  ctx.set('apiVersion', '1')
```

Comment:
Instead of a constant string, should we take from `.version`?

Reply:
So the idea of this is that only if this matches with the version defined in this PR -> https://github.com/rudderlabs/rudder-server/pull/380/files

With changing versions in transformer, we don't want an update in rudder-backend for every transformer version change.

We should look at this as a version that forces a change in rudder-backend.
That being said, instead of using a constant here, picking it up from somewhere else might make sense.

Reply:
Makes sense!